.github/workflows/ms.network.virtualnetworks.yml

```diff
@@ -28,21 +28,16 @@ on:
       - main
     paths:
       - '.github/actions/templates/**'
+      - '.github/workflows/template.module.yml'
       - '.github/workflows/ms.network.virtualnetworks.yml'
       - 'modules/Microsoft.Network/virtualNetworks/**'
       - 'utilities/pipelines/**'
       - '!utilities/pipelines/deploymentRemoval/**'
       - '!*/**/readme.md'
 
```

Comment:
General: I recall we cannot use the `:` character in the Azure DevOps pipeline names. Should we maybe use the opportunity to change the character to `-` for GitHub worklfow names to align the two?